### PR TITLE
Fixes security error around UUID in the delete confirmation code.

### PR DESCRIPTION
**Pull request by bseeger**
In theory this fixes the error seen here: https://github.com/codeforamerica/form-flow/security/code-scanning/1

Although, now I'm wondering if this would be fixed by simply making all the items either in the String.format() or concatenations, but not both.

### CODE REVIEW

**Comment by bseeger** [comment]
Doesn't appear to work. I'll try again tomorrow.

**Comment by spokenbird** [comment]
Curious what the security error here was?

**Comment by bseeger** [comment]
@spokenbird, it's this: https://github.com/codeforamerica/form-flow/security/code-scanning/1

File: src/main/java/formflow/library/ScreenController.java [hunk 1]
```diff
@@ -55,10 +57,12 @@
 @EnableAutoConfiguration
 @Slf4j
 @RequestMapping(ScreenController.FLOW)
+@Validated
 public class ScreenController extends FormFlowController {
 
   public static final String FLOW = "/flow";
   public static final String FLOW_SCREEN_PATH = "{flow}/{screen}";
+  private final String UUID_REGEX = "[0-9a-fA-F]{8}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{4}\\-[0-9a-fA-F]{12}";
```
Comment (bseeger) [inline src/main/java/formflow/library/ScreenController.java:65]: Move this to regex util file. 
Use this on other  spots where uuid is passed in as well, in this file.

**Comment by bseeger** [comment]
I expanded this PR to include all the spots where controllers take in a String uuid (submissionId, fileId).  Fixes related tests.

**Comment by coltborg** [comment]
@bseeger, Margaret and I discussed this PR. Bethany will be opening a new PR to fix this in a different way. I'm going to be bold and go ahead and close this PR.

**Pull request closed without merging**